diff --git a/node_modules/react-native-modalbox/index.js b/node_modules/react-native-modalbox/index.js
index 4742558..831c5f6 100644
--- a/node_modules/react-native-modalbox/index.js
+++ b/node_modules/react-native-modalbox/index.js
@@ -112,7 +112,7 @@ var ModalBox = createReactClass({
     return true
   },
 
-  componentWillMount: function() {
+  UNSAFE_componentWillMount: function() {
     this.createPanResponder();
     this.handleOpenning(this.props);
     // Needed for IOS because the keyboard covers the screen
@@ -129,7 +129,7 @@ var ModalBox = createReactClass({
     if (this.props.backButtonClose && Platform.OS === 'android') BackButton.removeEventListener('hardwareBackPress', this.onBackPress);
   },
 
-  componentWillReceiveProps: function(props) {
+  UNSAFE_componentWillReceiveProps: function(props) {
      if(this.props.isOpen != props.isOpen){
         this.handleOpenning(props);
      }
